src/webview/__tests__/generateInboundEventEditSequence-test.js

```diff
@@ -437,8 +439,12 @@ describe('messages -> piece descriptors -> content HTML is stable/sensible', ()
         state: eg.reduxStatePlus({
           streams: [...eg.plusReduxState.streams, stream1, stream2],
```

Comment:
This could be simplified, I think, by starting from `baseState` rather than use `eg.reduxStatePlus` directly.

Reply:
Hmm, now I'm wondering if the other uses of `eg.reduxStatePlus` in these tests should be switched to start from `baseState` too. … I guess many of them avoid an issue here by using `baseSingleMessage`, which has its stream ID fixed at `-1`. Which is cleverly outside the range that `makeStream` will ever choose.

The one awkward thing about that situation is that -1 isn't a valid stream ID at all. Possibly a good thing to do here would be for `makeStream` to never generate a stream ID in some range of small numbers, like never less than 10; and then tests can safely make up constant stream IDs within that range, and never flakily collide. We could do the same for user IDs and message IDs, too.

Reply:
> Possibly a good thing to do here would be for `makeStream` to never generate a stream ID in some range of small numbers, like never less than 10; and then tests can safely make up constant stream IDs within that range, and never flakily collide. We could do the same for user IDs and message IDs, too.

This sounds good; I'll include this in my next revision.

Reply:
> `PerAccountState` is an opaque type, so we can't write down a literal one in this file, which we'd want to do if making a state that starts from `baseState`.

Hmm yeah, I see.